aspnetcore/blazor/hybrid/static-files.md

```diff
@@ -0,0 +1,277 @@
+---
+title: ASP.NET Core Blazor Hybrid static files
```

Comment:
I guess this may be a marketing question, but do we need `ASP.NET Core` in this title? Should we somehow integrate `.NET MAUI` in there? I realize this is probably not something unique to this page, but just a thought.

Reply:
Yes, that's a requirement of all docs in the doc set per the higher-ups in the doc division.

> BH isn't exclusively .NET MAUI given WPF/WinForms

Readers looking for WPF/WinForms coverage may avoid this topic in search results if it's that specific. I don't think anyone can predict what the miss rate would be. My guess is that it would be a significant rate. I don't recommend titling it with .NET MAUI unless it also calls out WPF and WinForms. For example ...

> ASP.NET Core Blazor Hybrid static files in .NET MAUI, WPF, and Windows Forms apps

Reply:
**_Found it!_** 🔍 It was instituted back in early 2018 per ...

**Add technology names to doc titles and h1's to improve discoverability in search**
https://github.com/dotnet/AspNetCore.Docs/issues/3278

Reply:
> improve discoverability in search

Ahhh that makes sense. Thanks for the information!